packages/ata/package.json

```diff
@@ -26,6 +26,6 @@
     "jest": "^29.5.0"
   },
   "peerDependencies": {
-    "typescript": "^4.4.4"
+    "typescript": ">=4.4.4 <6.0.0"
```

Comment:
I wouldn't bother setting an upper bound; TypeScript doesn't generally follow semver in this way so 6 is just going to follow 5.9.

Reply:
Oh nice! Happy to change it.

Reply:
Updated